website/pages/docs/deployment/ecs.md

```diff
@@ -0,0 +1,379 @@
+  - `<REPLACE_CQ_BASE64_ENCODED_CONFIG>` : The base64 encoded version of the CloudQuery configuration file you created in Step 1.
+  - `<REPLACE_LOG_GROUP_NAME>` : The name of the CloudWatch log group you created in Step 4.
+  - `<REPLACE_AWS_REGION>` : The AWS region where you created the CloudWatch log group in Step 4.
+  - `<REPLACE_PREFIX_FOR_STREAM>` : The prefix you want to use for the CloudWatch log stream.
+  - `<REPLACE_TASK_FAMILY_NAME>` : The name of the task family you want to use.
+
+
+Once you have modified the `task-definition.json` file to include the correct values for your environment, you can register the task definition with AWS ECS using the following command:
+```bash
+
+aws ecs register-task-definition --cli-input-json file://task-definition.json
+
+```
+This command registers the task definition with AWS ECS and returns the task definition's ARN, which you will use in the next step when you run the task.
+
+## Step 7: Run the CloudQuery Task on ECS
+Now that the task definition is registered, it's time to run the CloudQuery task on ECS using the `aws ecs run-task` command.
+
+```bash
+aws ecs run-task \
+  --cluster <REPLACE_ECS_CLUSTER_NAME> \
+  --task-definition <REPLACE_TASK_ARN> \
+  --launch-type FARGATE \
+  --network-configuration 'awsvpcConfiguration={subnets=[<REPLACE_SUBNET_1>,<REPLACE_SUBNET_2>],securityGroups=[<REPLACE_SG_1>,<REPLACE_SG_2>]}'
+```
+Replace the following placeholders: 
+  - `<REPLACE_ECS_CLUSTER_NAME>` with the name of the ECS cluster you created in Step 2
+  - `<REPLACE_TASK_ARN>` with the ARN of the task definition you registered in Step 6
+  - `<REPLACE_SUBNET_1>` and `<REPLACE_SUBNET_2>` with the IDs of the subnets in which you want to run the task. You can specify any number of subnets that you want
+  - `<REPLACE_SG_1>` and `<REPLACE_SG_2>` with the IDs of the security groups for the task. You can specify any number of security groups that you want
+
+
+## Step 8: Schedule the Task to Run on a Regular Basis
+
+Now that you have a task that runs CloudQuery, you can schedule it to run on a regular basis using AWS EventBridge scheduler. An EventBridge schedule is able to start a task on a regular basis, but to do so it needs a role that it can assume which has the `ecs:RunTask` permission. In this step, you will create a role that has the required permissions and then you will create a schedule that will run the task on a regular basis.
+
+
+Create a file named `trust-policy.json` with the following contents:
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [{
+    "Version": "2012-10-17",
+    "Statement": [{
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "scheduler.amazonaws.com"
+      },
+      "Action": "sts:AssumeRole",
+      "Condition": {
+        "StringEquals": {
+          "aws:SourceArn": "arn:aws:scheduler:"<REPLACE_AWS_REGION>":"<REPLACE_AWS_ACCOUNT_ID>":schedule/default/"<REPLACE_SCHEDULE_NAME>""
```

Comment:
```suggestion
          "aws:SourceArn": "arn:aws:scheduler:<REPLACE_AWS_REGION>:<REPLACE_AWS_ACCOUNT_ID>:schedule/default/<REPLACE_SCHEDULE_NAME>"
```

Not sure this is the correct fix, but the current content is not a valid JSON file